Source/WebCore/rendering/svg/SVGTextLayoutEngine.h

```diff
@@ -90,6 +90,9 @@ class SVGTextLayoutEngine {
     float m_y;
     float m_dx;
     float m_dy;
+    float m_lastChunkStartPosition { 0.0f };
+    bool m_lastChunkHasTextLength { false };
+    bool m_lastChunkIsVerticalText { false };
```

Comment:
I prefer this style in general, but not needed in this patch to fix similar member vars.